### ISSUE TITLE
filename is too long(error=206,), when using gradle-scoverage plugin

### ISSUE THREAD

**Issue by AntonTkachev** [scoverage/gradle-scoverage#79]
Hello! 
I use your plugin version 2.3.
My build.gradle dropped on task 'testScoverage' with exception: 

> CreateProcess error=206, The filename or extension is too long.

I tried fix it and added task '[pathingJar](https://tuhrig.de/gradles-bootrun-and-windows-command-length-limit/)'. Something like this: 
```
testScoverage {
	dependsOn(pathingJar)
	classpath = files(pathingJar.archivePath)
}
```

But after it, task 'checkScoverage' drop, because couldn't find 'scoverage.xml'

If update to 2.4 version. I catch other exception on task 'testScoverage':

> What went wrong:
> Could not normalize path for file 'C:\Projects\myProject\build\scoverage\scoverage.measurements.*'.
> Invalid argument

Maybe you can help with it?

**Comment by maiflai** [comment]
Hi,

Sorry about this.

I think that your 2.3 issue is a variant of a core Gradle bug which has various workarounds (see https://github.com/maiflai/gradle-scalatest/issues/30).

I think the 2.4 issue is likely again a Gradle bug and specific to Windows. This plugin is tested automatically on Linux and manually on macOS, but unfortunately I don't have a Windows platform to test on. Do you have access to either of those platforms to confirm that the problem is isolated to Windows?

Thanks,
Stu

**Comment by AntonTkachev** [comment]
Sorry for late answer.

This case: https://github.com/maiflai/gradle-scalatest/issues/30#issuecomment-267450489 ,fix my problem on 2.3 version.

Thank you a lot.
